message_parser/src/test/java/cumulus_message_adapter/message_parser/AdapterUtilities.java

```diff
@@ -113,28 +87,27 @@ private static File createNewFileFromZipEntry(File destinationDir, ZipEntry zipE
      * @throws IOException
      */
     private static void unzipFile(String fileZip, String dest) throws IOException {
```

Comment:
The current implementation doesn't preserve the executable permission, and `unzip` command extracts and preserve file permissions.